eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,7 +1,10 @@
 param (
   $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
   $RootDirectory, # ideally $(Build.SourcesDirectory)
-  $VsoVariable = "" # target devops output variable
+  $AuthToken,
+  $VsoOwningUsers = "", # target devops output variable
+  $VsoOwningTeams = "",
+  $VsoOwningLabels = ""
```

Comment:
Labels parameter is still here. BTW I was fine with you keeping the labels functionality here as that might become useful at some point as well.